subprojects/packagefiles/libffi/meson.build

```diff
@@ -95,7 +95,7 @@ elif host_cpu_family.startswith('alpha')
 elif host_cpu_family.startswith('arc')
   target = 'ARC'
   asm_sources = ['arcompact.S']
-elif host_cpu_family == 'arm' and is_windows
+elif host_cpu_family == 'arm' and host_system in ['cygwin', 'windows']
```

Comment:
Same here, does cygwin even run on arm?

Reply:
That's a translation of https://github.com/libffi/libffi/blob/master/configure.host#L9, I have no idea if it makes sense or not...

Reply:
... I guess it doesn't hurt since it's an impossible situation but would be good to get the maintainer to confirm